[docs revamp] Data Migration (#390)
dgraph-io/dgraph-docs: content/migration/loading-csv-data.md

### Before you begin

* Install [`csv2json`](https://www.npmjs.com/package/csv2json) conversion tool.
* Install `jq` a lightweight and flexible command-line JSON processor.
* Connect the Dgraph instance to Ratel for queries, mutations and visualizations.

#### Example 1

1. Create a `names.csv` file with these details:

    ```csv
    Name,URL
    Dgraph,https://github.com/dgraph-io/dgraph
    Badger,https://github.com/dgraph-io/badger
    ```

2. Change to the directory that conntains the `names.csv` file and convert it to `names.json`:

    ```sh
    $ csv2json names.csv --out names.json
    ```
3. To prettify a JSON file, use the jq '.' command:

    ```sh
    $ cat names.json | jq '.'
    ```
    The output is similar to:
    ```sh
    [
      {
        "Name": "Dgraph",
        "URL": "https://github.com/dgraph-io/dgraph"
      },
      {
        "Name": "Badger",
        "URL": "https://github.com/dgraph-io/badger"
      }
    ]
    ```

>     This JSON file can be loaded into Dgraph using the programmatic clients. This follows
>     the [JSON Mutation Format]({{< relref "mutations/json-mutation-format.md" >}}).
>     Note that each JSON object in the list is assigned a unique UID because the `uid` field is omitted.

Reviewer: ```suggestion
    This JSON file follows
    the [JSON Mutation Format]({{< relref "mutations/json-mutation-format.md" >}}), it can be loaded into Dgraph using [Dgraph Live Loader]({{< relref "/deploy/fast-data-loading/live-loader.md" >}}) , [Dgraph Bulk Loader]({{< relref "/deploy/fast-data-loading/bulk-loader.md" >}}) or the programmatic clients.
```
